src/openzaak/tests/management/test_generate_data.py

```diff
@@ -38,10 +43,28 @@ def test_generate_data_yes(self, m):
                         "procestermijnWeergave": "Nihil",
                         "bewaartermijn": "P10Y",
                         "toelichting": "Invoering nieuwe werkwijze",
+                        "waardering": "vernietigen",
                     }
                 ],
             },
         )
+        m.get(
+            f"{config.api_root}resultaattypeomschrijvingen",
+            json=[
+                {
+                    "url": f"{config.api_root}resultaattypeomschrijvingen/ce8cf476-0b59-496f-8eee-957a7c6e2506",
+                    "omschrijving": "Afgebroken",
+                    "definitie": "Afgebroken",
+                    "opmerking": "",
+                },
+                {
+                    "url": f"{config.api_root}resultaattypeomschrijvingen/7cb315fb-4f7b-4a43-aca1-e4522e4c73b3",
+                    "omschrijving": "Afgehandeld",
+                    "definitie": "Afgehandeld",
+                    "opmerking": "",
+                },
+            ],
+        )
```

Comment:
I dont think this gets called. Is it supposed to be?

Reply:
I commented it out and the test passes

Reply:
Hmmm I've commented it and this test fails for me in this line:
https://github.com/open-zaak/open-zaak/blob/fe5b36e72843f94f5d5aea23438ef178f2e593cd/src/openzaak/management/commands/generate_data.py#L263
 
To fill in `resultaattype.resultaattypeomschrijving` we add extra request to Selectielijst API in the `generate_data` command, because we need the url from there.

Reply:
It should be fine then. IDK why it works locally